sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_document_model_administration_client.py

```diff
@@ -141,21 +196,13 @@ def begin_build_document_model(
             setting up a training data set, see: https://aka.ms/azsdk/formrecognizer/buildtrainingset.
         :keyword str model_id: A unique ID for your model. If not specified, a model ID will be created for you.
         :keyword str description: An optional description to add to the model.
-        :keyword str prefix: A case-sensitive prefix string to filter documents in the blob container url path.
-            For example, when using an Azure storage blob URI, use the prefix to restrict sub folders.
-            `prefix` should end in '/' to avoid cases where filenames share the same prefix.
-        :keyword str file_list: Path to a JSONL file within the container specifying a subset of
-            documents for training.
```

Comment:
Last I checked, we don't get the documentation from the overloads on the docs sites so we might need to keep the docs here otherwise they won't show up :(

Reply:
:( that's sad...let me check the docs I'll revert this if that's the case

Reply:
Here's a similar overload in translation where it doesn't pick up the overload docstrings: https://azuresdkdocs.blob.core.windows.net/$web/python/azure-ai-translation-document/latest/azure.ai.translation.document.html#azure.ai.translation.document.DocumentTranslationClient.begin_translation

Reply:
Sigh, I have sadly reverted that change. Thanks for pointing it out!